ViteMaDose/Resources/fr.lproj/Localizable.strings

```diff
@@ -0,0 +1,44 @@
+/* 
+  Localizable.strings
+  ViteMaDose
+
+  Created by Victor Sarda on 24/04/2021.
+  
+*/
+
+// Home
+"home.main_title" = "Trouvez une dose de vaccin facilement et rapidement";
+"home.main_title.first_highlighted_text" = "facilement";
+"home.main_title.second_highlighted_text" = "rapidement";
+"home.select_county" = "Sélectionner un département...";
+"home.recent_search" = "Recherche récente";
+"home.last_stats" = "Dernières statistiques";
+"home.open_map" = "Ouvrir la carte des lieux ";
```

Comment:
Change requested. Consistency: Remove the trailing space and add "+ String.space" when needed.